src/starcheck.pl

```diff
@@ -331,8 +331,16 @@
 
 Ska::Starcheck::Obsid::set_config($config_ref);
 
-
-
+my $MSF_ENABLED;
```

Comment:
It isn't strictly required, but if we use string comparison it needs to be

$bs[0]->{date} gt ''2016:102:00:00:00.000'

and I *think* there are locale specific sort orders when you get into
string comparisons, so the numerics might be safer.  Granted, we are
probably using that string comparison in other places in the code.



On Mon, Apr 11, 2016 at 6:57 AM, Tom Aldcroft <notifications@github.com>
wrote:

> In src/starcheck.pl
> <https://github.com/sot/starcheck/pull/142#discussion_r59186399>:
>
> > @@ -331,8 +331,16 @@
> >
> >  Ska::Starcheck::Obsid::set_config($config_ref);
> >
> > -
> > -
> > +my $MSF_ENABLED;
>
> I think you don't need the date2time:
>
> $bs[0]->{date} > '2016:102:00:00:00.000'
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/sot/starcheck/pull/142/files/ef4eac5188d6f64c4ae5ec6029d625fbc9675297#r59186399>
>